CHANGES.md

```diff
@@ -1,3 +1,33 @@
+# grmtools 0.13.0 (XXXX-XX-XX)
+
+## Breaking changes
+
+* `cfgrammar::yacc::grammar::YaccGrammarSpan` now returns `Option<Span>` rather
+  than `Option<&Span>`.
+
+* `cfgrammar::yacc::ast::{Production, Symbol}` no longer derive `Eq` and
+  `PartialEq`. Since both now carry a `Span`, it's easy to confuse "two
+  {productions, symbols} have the same name" with "at the same place in the
+  input file."
+
+
+## Deprecations
+
+* `Span` has moved from `lrpar` to `cfgrammar`. The import is still available
+  via `lrpar` but is deprecated (though due to
+  https://github.com/rust-lang/rust/issues/30827 this unfortunately does not
+  show as a formal deprecation warning).
+
+## New features
+
+* Improved error messages, with various parts of grammar and lexer files now
+  carrying around `Span` information to help pinpoint errors.
```

Comment:
I believe that is it if we are referring to field additions here, but i'd like to do a more thorough look through the history tomorrow.